src/main/java/com/hyunec/cosmicbaseballinit/controller/HitterGameInterfaceImpl.java

```diff
@@ -2,61 +2,65 @@
 
 import com.hyunec.cosmicbaseballinit.config.ReadMessageYml;
 import com.hyunec.cosmicbaseballinit.service.lv1.HitterGameService;
+import com.hyunec.cosmicbaseballinit.vo.hitterGame.HitterResult;
 import com.hyunec.cosmicbaseballinit.vo.hitterGame.PitchProbabilitySettingVo;
 import com.hyunec.cosmicbaseballinit.vo.hitterGame.HittingParamVo;
 import com.hyunec.cosmicbaseballinit.vo.hitterGame.PitchResult;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.stereotype.Component;
 
 import java.util.Map;
 
-@RestController
+@Component
 @RequiredArgsConstructor
 @Slf4j
-public class GameController {
+public class HitterGameInterfaceImpl implements HitterGameInterface{
 
     private final HitterGameService gameService;
     private final ReadMessageYml readMessageYml;
 
-    @GetMapping("/game/setting")
     public String gameSetting(){
         gameService.setHitGameProbability();
         return readMessageYml.getSettingFinished();
     }
 
-    @PostMapping("/game/setting")
-    public String gameSetting(@RequestBody PitchProbabilitySettingVo pitchProbabilitySettingVo){
+    public String gameSetting(PitchProbabilitySettingVo pitchProbabilitySettingVo){
         gameService.setHitGameProbability(pitchProbabilitySettingVo);
         return readMessageYml.getSettingFinished();
     }
 
     // 타구 확률을 클라이언트로 부터 입력 받아서 타구
-    @PostMapping("/game/hitting")
-    public String hitting(@RequestBody HittingParamVo hittingParamVo) throws Exception {
+    public String hitting(HittingParamVo hittingParamVo) throws Exception {
         String hittingResult = gameService.hitting(
                 hittingParamVo.getPitchResultRandomDouble(),
                 hittingParamVo.getHitterResultRandomDouble());
         return hittingResult;
     }
 
     // 타구 확률을 랜덤 값으로 하여 타구
-    @GetMapping("/game/hitting/random")
     public String hitting() throws Exception {
         String hittingResult = gameService.hitting(
                 Math.random(),
                 Math.random());
         return hittingResult;
     }
 
-    @GetMapping("/game/initScore")
     public void initScore() {
         gameService.initGameScore();
     }
 
-    @GetMapping("/game/hitterScore")
     public Map<PitchResult, Integer> hitterScore(){
         return gameService.getScores();
     }
 
+    @Override
+    public Boolean isHitterGameEnd() {
```

Comment:
```suggestion
    public Boolean isEnd() {
```
`nit` 인터페이스를 활용하고 클래스 네이밍에 `HitterGame` 이 이미 들어가 있기에 생략하는 것도 방법입니다.